app/views/orga/teams/show.html.slim

```diff
@@ -79,7 +79,7 @@ nav.btn-group
   ul.conferences
     - @team.members.each do |user|
       - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season, true).join(',')}".html_safe
+        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(',')}".html_safe
```

Comment:
Sure ✌️ 

The easiest way would have been to just append the `join` etc. to the already existing method:
```ruby
def links_to_conferences(conferences)
  conferences.map do |conference|
    text = conference.name
    extra = [conference.location, format_conference_date(conference.starts_on, conference.ends_on)].reject(&:blank?).join(' – ')
    text += " (#{extra})" unless extra.blank?
    link_to(text, conference)
  end.join(',').html_safe
end
```
But yeah, there's probably a prettier way.